cmd/eval.go

```diff
@@ -22,16 +22,6 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var arguments = struct {
-	policyConfiguration string
-	imageRef            string
-	publicKey           string
-	rekorURL            string
-	strict              bool
-}{
-	rekorURL: "https://rekor.sigstore.dev/",
-}
-
```

Comment:
I don't quite understand how is this compiling, the code below uses `arguments`

Reply:
I think it's because it's the same package.

Reply:
Right I missed that these got moved to `root.go`, can we keep them here instead, the intent is that these arguments are for the `eval` functionality, trying to share this struct between two sub-comands will lead to unexpected behavior (e.g. nil reference) on trying to access a field that was not bound.

Reply:
We can, issue is that eval and verify are under the `cmd` package, any `var` will be accessible for both

Reply:
If that the case, we prefix them within the struct to have `arguments.eval.imageRef` wdyt ?

Reply:
I think we should do something that @robnester-rh suggested and implemented in the validate pipeline PR: https://github.com/hacbs-contract/ec-cli/pull/15/files#diff-6701a60d95525f3d714ff656fdc86ef5dc9753df9e79b60f0a9a7d2be4d884c6R34-R48

But we can refactor this after this PR to have this done first...

Reply:
Yup, made the change

Reply:
Moved it to eval, and got rid of verify